Cargo.toml

```diff
@@ -2,9 +2,15 @@
 name = "sensirion-i2c-rs"
 version = "0.1.0"
 authors = ["Raphael Nestler <raphael.nestler@sensirion.com>"]
+description = "Common functionality for I2C based sensors from Sensirion"
+readme = "README.md"
+keywords = ["I2C", "Sensirion"]
+categories = ["no-std", "hardware-support", "embedded"]
+license = "BSD-3-Clause"
+repository = "https://github.com/Sensirion/sensirion-i2c-rs"
 edition = "2018"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-embedded-hal = "1.0.0-alpha.1"
+embedded-hal = "*"
```

Comment:
This won't be allowed when publishing to Crates.io... You have to specify a version.

Reply:
Thanks, good catch :slightly_smiling_face:

Reply:
Fixed.

Reply:
(My usual approach to add the most recent version of a crate is to first just specifiy `*` in `Cargo.toml`, then check `Cargo.lock` which version that is and update `Cargo.toml` with that version. I should probably rethink that approach :wink:)

Reply:
Mine is opening `docs.rs/<crate-name>` :smile: